src/mopeka.c

```diff
@@ -365,29 +380,29 @@ int mopeka_handle_mfg(const bdaddr_t *addr, const uint8_t *buf, int len)
 	hwid = buf[0];
 
 	switch (hwid) {
-	case HW_ID_LPG:
-		type = "LPG";
+	case HW_ID_PRO:
+		type = "Pro";
 		break;
-	case HW_ID_H2O:
-		type = "H2O";
+	case HW_ID_PRO_H2O:
+		type = "Pro H2O";
 		break;
-	case HW_ID_P200:
-		type = "Pro200";
+	case HW_ID_PRO_200:
+		type = "Pro 200";
 		break;
-	case HW_ID_PPB:
-		type = "PPB";
+	case HW_ID_PRO_PLUS_BLE:
+		type = "Pro+ BLE";
 		break;
-	case HW_ID_PPC:
-		type = "PPC";
+	case HW_ID_PRO_PLUS_CELL:
+		type = "Pro+ Cell";
 		break;
-	case HW_ID_TDB:
-		type = "TDB";
+	case HW_ID_TOPDOWN_BLE:
+		type = "Topdown BLE";
```

Comment:
These descriptions can't be used because the string is used to build the name of the device in Victron's platform and device names have a very strict length constraint.  They shortened the descriptors I used that time and I had already made them pretty short.  My goal was to at least be descriptive enough that you can tell one model from another, rather than the verbose names.